VideoWeb/VideoWeb.Common/Caching/DistributedHearingLayoutCache.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.Extensions.Caching.Distributed;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using VideoApi.Contract.Requests;
+
+namespace VideoWeb.Common.Caching
+{
+    public class DistributedHearingLayoutCache : IHearingLayoutCache
+    {
+        private readonly IDistributedCache _distributedCache;
+        private readonly string _cachePrefix = "layout_";
+
+        public DistributedHearingLayoutCache(IDistributedCache distributedCache)
+        {
+            _distributedCache = distributedCache;
+        }
+
+        public async Task Write(Guid conferenceId, HearingLayout layout)
+        {
```

Comment:
It's not a must but considering how much of this code is repetitive between each of the cache stores, do you think there's any way you could minimise the duplication?